examples/text_to_image/train_text_to_image_mlflow.py

```diff
@@ -61,41 +63,39 @@
 logger = get_logger(__name__, log_level="INFO")
 
 DATASET_NAME_MAPPING = {
-    "lambdalabs/pokemon-blip-captions": ("image", "text"),
+    "pokemon-blip-captions": ("image", "text"),
 }
 
 
 def get_num_parameters(model):
-  num_params = 0
-  for param in model.parameters():
-    num_params += param.numel()
-  # in million
-  num_params /= 10**6
-  return num_params
+    num_params = 0
+    for param in model.parameters():
+        num_params += param.numel()
+    # in million
+    num_params /= 10**6
+    return num_params
+
 
 def log_validation(vae, text_encoder, tokenizer, unet, args, accelerator, weight_dtype, epoch):
     logger.info("Running validation... ")
-
     pipeline = StableDiffusionPipeline.from_pretrained(
         args.pretrained_model_name_or_path,
-        vae=accelerator.unwrap_model(vae),
-        text_encoder=accelerator.unwrap_model(text_encoder),
+        vae=vae,
+        text_encoder=text_encoder,
         tokenizer=tokenizer,
         unet=accelerator.unwrap_model(unet),
```

Comment:
Why `unet` is unwrapped but `vae` and `text_encoder` are not? 
Imo, on MAF, whether a module is wrapped or unwrapped does not matter, so why did you make this change?

Reply:
Since in this script, `vae` and `text_encoder` are kept unchanged (`.requires_grad_(False)`) while `unet` is trained. I just want to ensure the desired behavior cuda gpu